lib/matplotlib/tests/test_figure.py

```diff
@@ -216,6 +217,28 @@ def test_figaspect():
     assert h / w == 1
 
 
+@cleanup
+def test_stack_remove():
+    '''
+    Before fixing the bug, doing the following will give you following error:
```

Comment:
sure, I added the docstring because I got people asking me what is the test for.